graylog2-server/src/main/java/org/graylog2/contentpacks/ContentPackAble.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.contentpacks;
+
+public interface ContentPackAble<T> {
+    Object toContentPackEntity();
```

Comment:
The only case where it can be null was with the FallbackConfig, and now I throw there an `UnsupportedOperationException` which is what the other functions do as well.